packages/core/test/test_util/common_verifications.js

```diff
@@ -0,0 +1,143 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
+
+const expectExactlyOneMatching = require('./expectExactlyOneMatching');
+const constants = require('../../src/index').tracing.constants;
+
+/**
+ * @typedef {import('../../src/tracing/cls').InstanaBaseSpan} InstanaBaseSpan
+ */
+
+/**
+ * @typedef {Object} HttpRootEntryOptions
+ * @property {Chai.ExpectStatic} expect
+ * @property {Array.<InstanaBaseSpan>} spans
+ * @property {string} apiPath
+ * @property {string} pid
+ * @property {Array.<(span: InstanaBaseSpan) => void>} [extraTests]
+ */
+
+/**
+ * @typedef {Object} ExitSpanOptions
+ * @property {Chai.ExpectStatic} expect
+ * @property {string} spanName
+ * @property {Array.<InstanaBaseSpan>} spans
+ * @property {InstanaBaseSpan} parent
+ * @property {boolean} withError
+ * @property {string} pid
+ * @property {Array.<(span: InstanaBaseSpan) => void>} [extraTests]
+
+ */
+
+/**
+ * @typedef {Object} HttpExitOptions
+ * @property {Chai.ExpectStatic} expect
+ * @property {Array.<InstanaBaseSpan>} spans
+ * @property {InstanaBaseSpan} parent
+ * @property {string} pid
+ * @property {Array.<(span: InstanaBaseSpan) => void>} [extraTests]
+ */
+
+/**
+ * @param {HttpRootEntryOptions} options
+ * @returns {InstanaBaseSpan}
+ */
+// eslint-disable-next-line
+exports.verifyHttpRootEntry = function verifyHttpRootEntry({ expect, spans, apiPath, pid, extraTests }) {
```

Comment:
Meant to ask that in the last review, but I forgot: Why do these functions receive `expect` as a parameter. It should be the same expect function all the time. Why do we not just import `expect` here in this file?

Reply:
That completely escaped from my mind. I will do this change :D
Thanks!